stacks/openfaas/upgrade.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add openfaas https://openfaas.github.io/faas-netes/
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="openfaas"
+CHART="openfaas/openfaas"
+NAMESPACE="openfaas"
+
+if [ -z "${MP_KUBERNETES}" ]; then
+    # use local version of values.yml
+    ROOT_DIR=$(git rev-parse --show-toplevel)
+    values="$ROOT_DIR/stacks/openfaas/values.yml"
+else
+    # use github hosted master version of values.yml
+    values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/openfaas/values.yml"
+fi
+
+helm upgrade "$STACK" "$CHART" \
+--namespace "$NAMESPACE" \
+--values "$values" \
+
+helm upgrade ingress-nginx \
+  --namespace "$NAMESPACE"
+
+helm upgrade cert-manager \
+  --namespace "$NAMESPACE"
```

Comment:
I seem to be ggetting this output when I run this script locally. Seems like your two helm commands are incomplete?

```console
➜  marketplace-kubernetes-openfaas_stack git:(master) ✗ ./stacks/openfaas/upgrade.sh
"openfaas" already exists with the same configuration, skipping
Release "openfaas" has been upgraded. Happy Helming!
NAME: openfaas
LAST DEPLOYED: Wed Mar  9 01:03:11 2022
NAMESPACE: openfaas
STATUS: deployed
REVISION: 4
TEST SUITE: None
NOTES:
To verify that openfaas has started, run:

  kubectl -n openfaas get deployments -l "release=openfaas, app=openfaas"
To retrieve the admin password, run:

  echo $(kubectl -n openfaas get secret basic-auth -o jsonpath="{.data.basic-auth-password}" | base64 --decode)
Error: "helm upgrade" requires 2 arguments

Usage:  helm upgrade [RELEASE] [CHART] [flags]
```

Reply:
ingress-nginx and cert-manager has been removed and now the upgrade script will handle only openfaas app